platforms/software/reporting/src/main/java/org/gradle/api/reporting/internal/SimpleReport.java

```diff
@@ -68,9 +62,7 @@ public void setDestination(File file) {
         getOutputLocation().fileValue(file);
     }
     @Override
-    public OutputType getOutputType() {
-        return outputType;
-    }
+    public abstract Property<OutputType> getOutputType();
```

Comment:
💅 Probably not that important since it's internal type, but should this be just a Provider?